stock_picking_responsible/migrations/13.0.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Coop IT Easy SC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
+    cr.execute(
+        """
+        ALTER TABLE stock_picking ADD IF NOT EXISTS legacy_13_0_responsible_id int;
+        UPDATE stock_picking SET legacy_13_0_responsible_id = responsible_id;
```

Comment:
So if I'm reading this right, you're preserving the legacy column outside the ORM, then `odoo --update` removes the responsible_id field? Doesn't that suggest that you would need a v14 migration to delete that column?

Alternately:
1. Delete the pre-migration.
2. Add responsible_id field to `stock.picking` model, but not to any views and label it "Legacy Responsible".

This preserves the legacy data, with the easy option for the administrator to delete it by uninstalling the module. It also saves some database churn by not adding an equivalent column and copying values.